src/metabase/driver/generic_sql.clj

```diff
@@ -172,8 +172,8 @@
                                                (-fetch-page (inc page-num)))))))]
     (fetch-page 0)))
 
-(defn- table-rows-seq [_ database table-name]
-  (k/select (korma-entity database {:table-name table-name})))
+(defn- table-rows-seq [_ database table]
```

Comment:
as a function i think it makes more sense to require a `:schema` to also be included because only getting a table name is not adequate in many cases.  i can update the docs though.